buy-resell/products/3rd-party/datev-meinfiskal.md

```diff
@@ -1,76 +1,97 @@
 ---
 slug: /posdealers/buy-resell/products/3rd-party/datev-meinfiskal
 title: DATEV MeinFiskal
----
+---fiskaltrust.Portal
 # DATEV MeinFiskal
 
 :::info summary
 
-After reading this, you will know how the integration of DATEV MeinFiskal works and how to set it up.
+After reading this, you will understand the benefits for PosOperators of connecting their fiskaltrust.Account to _DATEV MeinFiskal_ and how PosDealers can set up the integration.
 
 :::
 
 :::caution Austria / France
 
-As DATEV MeinFiskal is only available in Germany, this tutorial does not apply to Austria or France.
+As _DATEV MeinFiskal_ is only available in Germany, this tutorial does not apply to Austria or France.
+
+Please note that the included links in this section lead to DATEV, which only keeps its documentation in German.
 
 :::
 
- "DATEV MeinFiskal" is an integral part of the "fiskaltrust.Sorglos" product bundle.The data is transferred from the fiskaltrust.Portal via an automated interface to the "[DATEV MeinFiskal](https://www.meinfiskal.de/)" platform, so the included services "DATEV Kassenarchiv online" and the "DATEV Kassenbuch-Schnittstelle" can be used.
+  [DATEV](https://www.datev.de/) offers fiscalization solutions for companies in Germany. The solutions for electronic cash management of DATEV meet all legal requirements.   
+_[DATEV MeinFiskal](https://www.meinfiskal.de/)_ is an open cloud platform hosted by DATEV. fiskaltrust, PosCreators and providers of TSE (technical security equipment) joined this platform.  

```

Comment:
It does not feel right to start a sentence in lowercase. Does it make sense to change the order if we want to emphasize lowercase in "fiskaltrust"? I.e. PosCreators, fiskaltrust and providers...